sodacitylabs/boring-framework
# support node 12

---
name: Pull Request
about: Ask for code to be merged into master
---

**Describe your Code**
Several changes to support Node12: bumping packages, the node version, and changing how the action regex works to support private fields. Also will help with supporting jsdoc comments

**Expected behavior**
Using node12 and going through the Getting Started guide works as well as adding private variables to controllers

**How to test**
Steps to test and verify the code works as expected:
1. Go to through the getting started guide
2. be sure to add private variables to make sure that the string replace is working
